### PR TITLE
Cleaning up adb forwards when we finish debugging

### LINKED ISSUE #23: Adb port forwarding persists after android debugging session ends.
If you debug for android and then try to debug for iOS without using a different `port` setting in launch.json, you'll hit an issue where the iOS code path attempts to attach to the android port and gets confused. For now we recommend that you use different ports or close the android emulator, but in future we should stop the adb port forwarding when the debug session terminates.

### PR DESCRIPTION
**Pull request by MSLaguana**
Fixes #23

### CODE REVIEW

File: src/debugger/cordovaDebugAdapter.ts [hunk 1]
```diff
@@ -154,7 +163,12 @@ export class CordovaDebugAdapter extends WebKitDebugAdapter {
                 // Configure port forwarding to the app
                 let forwardSocketCommandArguments = ['-s', targetDevice, 'forward', `tcp:${attachArgs.port}`, `localabstract:webview_devtools_remote_${pid}`];
                 this.outputLogger('Forwarding debug port');
-                return execCommand('adb', forwardSocketCommandArguments, errorLogger);
+                return execCommand('adb', forwardSocketCommandArguments, errorLogger).then(() => {
```
Comment (nisheetjain) [inline src/debugger/cordovaDebugAdapter.ts]: lets move this to a separate function

Comment (MSLaguana) [inline src/debugger/cordovaDebugAdapter.ts]: Done

Comment (nisheetjain) [inline src/debugger/cordovaDebugAdapter.ts]: hmm, now I understood why you were doing what you were doing :)

**Comment by nisheetjain** [comment]
:+1:

File: src/debugger/cordovaDebugAdapter.ts [hunk 2]
```diff
@@ -21,6 +21,7 @@ import {TelemetryHelper} from '../utils/telemetryHelper';
 
 export class CordovaDebugAdapter extends WebKitDebugAdapter {
     private outputLogger: (message: string,  error?: boolean) => void;
+    private adbPortForwardinginfo: {targetDevice: string, port: number};
```
Comment (meenakb) [inline src/debugger/cordovaDebugAdapter.ts]: Future-proof this by pascal-casing this - adbPortForwardingInfo

Comment (MSLaguana) [inline src/debugger/cordovaDebugAdapter.ts]: Thanks for catching this, done!

**Comment by meenakb** [comment]
:+1:

**Pull request merged**